<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchForm</class>
 <widget class="QDialog" name="SearchForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>358</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="L_Search_MainForm">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Търсене </string>
   </property>
  </widget>
  <widget class="QLabel" name="L_search_RegNumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>98</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Търсене по 
 Рег. Номер</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="L_search_Klient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>70</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Търсене 
 клиент</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="L_search_Auto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>96</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Търсене 
 автомобил</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="L_search_Data">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>70</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Търсене 
 по дата</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_StartSearch">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>80</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Търсене</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Combo_SearchAuto">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Автомобил</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="Combo_Search_Klient">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Клиент</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="Combo_Search_RegNumber">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Рег. Номер</string>
    </property>
   </item>
  </widget>
  <widget class="QCalendarWidget" name="Calendar_Search">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>248</width>
     <height>183</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Search_Back">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>290</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
